app/institutions/dashboard/-components/institutional-users-list/component.ts

```diff
@@ -133,11 +140,9 @@ export default class InstitutionalUsersList extends Component<InstitutionalUsers
 
     @tracked selectedColumns: string[] = this.columns.filter(col => col.selected).map(col => col.key);
     // Private properties
-    @tracked department = this.intl.t('institutions.dashboard.select_default');
-    @tracked sort = 'user_name';
     @tracked hasOrcid = false;
     @tracked totalUsers = 0;
-
+    orcidUrlPrefix = 'https://orcid.org/'; // Move to config?
```

Comment:
I don't _think_ that's something that's likely to need to change per environment, so probably doesn't need to be configurable.